stdpopsim/models.py

```diff
@@ -137,6 +137,176 @@ def asdict(self):
             "migration_matrix": self.migration_matrix,
             "demographic_events": self.demographic_events}
 
+# ###############################################################
+# UNDER CONSTRUCTION - GROUND TRUTH N_t
+# ###############################################################
+
+    # TODO Rename
+    def get_Ne_through_time_single_pop(self, end, num_steps=10):
```

Comment:
vote to rename this `population_size_trajectory()`

Reply:
And, vote to call this "population size" not "Ne" in the docs.

Reply:
agreed